### PR TITLE
Fix: Snackbar Inside Charges Dialog #654

### PR DESCRIPTION
**Pull request by aj019**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x ] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

After fix:
ezgif com-video-to-gif

### CODE REVIEW

**Comment by therajanmaurya** [comment]
This should not show try again. it should show a warning that amount can not be empty.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/chargedialog/ChargeDialogFragment.java [hunk 1]
```diff
@@ -198,7 +197,8 @@ public void showChargesCreatedSuccessfully(Charges changes) {
 
     @Override
     public void showFetchingError(String s) {
-        Toaster.show(rootView, s);
+        Toast.makeText(getActivity(), s , Toast.LENGTH_LONG)
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/chargedialog/ChargeDialogFragment.java]: This should not be a toast. It should be a snackbar attached to the background view to conform to the theme.

**Comment by tarun0** [comment]
@aj019 Please comment within a week if you can fix it. No updates will lead to the PR being closed.

**Comment by tarun0** [comment]
Closed for lack of updates.

**Pull request closed without merging**